### ISSUE TITLE
ignore videos that were manually removed from playlist AND later put back

### ISSUE THREAD

**Issue by ChrisIdema** [ninsky/recovermyvideotracker#22]
I moved some videos to a different playlist. The showed up as "recovered". Since I cannot unmark them as recovered I put them back into the original playlist. It still shows up as recovered. So this tool doesn't look if the video is at a different position in the playlist.

**Comment by ninsky** [comment]
Hello ChrisIdema

Thanks for your input! Currently I'm reworking the whole application logic. Thanks for letting me know about this issue. Will implement a fix and hope to roll the fix out within the next 3-4 weeks.

I will leave this issue open till the fix is on production.

BR
Nino

**Comment by ninsky** [comment]
Hello ChrisIdema

I've just checked this use case with my new application logic (not deployed yet), which **should** also corresponds to the current application logic.

As it is possible to save the same video on multiple playlists, RecoverMy.Video consideres these videos as unique. That means if you remove the video from Playlist A to Playlist B, it will be considered as "recovered" on Playlst A. If you move it back from B to A, it will be considered as "recovered" on Playlist B (but not on A).

As the application is not able to distinguish if a playlist video was removed by YouTube or yourself, there's no better solution for this case.

BR
Nino
